main_app/models.py

```diff
@@ -37,15 +37,15 @@ class Status(models.IntegerChoices):
   discount_ratio_applied = models.DecimalField(max_digits=4, decimal_places=2, blank=True, validators=[MinValueValidator(0), MaxValueValidator(1),], help_text=discount_ratio_applied_help_text)
 
   def get_readable_status(self):
-    if self.status == 0:
+    if self.status == self.Status.CURRENT:
```

Comment:
@bernallium  - is there a reason that the word status is capitalized on the right side of the equation? (lines 40 -48)?

Reply:
The status codes are constants defined in the Sratus class within the Cart model. It's just best practice to name them IN ALL_CAPS_SNAKE_CASING.